code/game/gamemodes/cm_self_destruct.dm

```diff
@@ -187,6 +187,287 @@ var/global/datum/authority/branch/evacuation/EvacuationAuthority //This is initi
 		if(lifeboat && lifeboat.available)
 			lifeboat.send_to_infinite_transit()
 
+//unlocks Manual Launch
+/datum/authority/branch/evacuation/proc/lifeboat_unlock_manual_launch()
+	for(var/obj/structure/machinery/lifeboat_bolt/target/bolt in machines)
+		bolt.unlock()
+
+////BOLT CONTROLS/// for lifeboat manual launch
+#define LOCKED		0
+#define UNLOCKED	1
+#define LAUNCHING	2
+#define LAUNCHED	3
+
+#define AIMING	1
+#define AIMED	2
+#define FIRING	3
+
+#define PRIMED_TEMP		1
+#define PRIMED_PERMA	2
```

Comment:
```suggestion
#define LOCKED 0
#define UNLOCKED 1
#define LAUNCHING 2
#define LAUNCHED 3

#define AIMING 1
#define AIMED 2
#define FIRING 3

#define PRIMED_TEMP 1
#define PRIMED_PERMA 2
```
do this or be careful with mixed tab/space indentation because the linters hate them (https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#tabs-not-spaces)